test/e2e/network/service.go

```diff
@@ -2617,3 +2617,33 @@ func checkReachabilityFromPod(expectToBeReachable bool, timeout time.Duration, n
 	})
 	framework.ExpectNoError(err)
 }
+
+// proxyMode returns a proxyMode of a kube-proxy.
+func proxyMode(f *framework.Framework) (string, error) {
+	pod := &v1.Pod{
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      "kube-proxy-mode-detector",
+			Namespace: f.Namespace.Name,
+		},
+		Spec: v1.PodSpec{
+			HostNetwork: true,
+			Containers: []v1.Container{
+				{
+					Name:  "detector",
+					Image: framework.AgnHostImage,
+					Args:  []string{"pause"},
+				},
+			},
+		},
+	}
+	f.PodClient().CreateSync(pod)
+	defer f.PodClient().DeleteSync(pod.Name, &metav1.DeleteOptions{}, framework.DefaultPodDeletionTimeout)
+
+	cmd := "curl -q -s --connect-timeout 1 http://localhost:10249/proxyMode"
+	stdout, err := framework.RunHostCmd(pod.Namespace, pod.Name, cmd)
+	if err != nil {
+		return "", err
+	}
+	framework.Logf("ProxyMode: %s", stdout)
```

Comment:
nit: How about changing to "proxyMode" according to function name?

Reply:
nice catch, will do that.
Thanks for your review :-)